src/drivers/general/queries/experimental/IrqlNotUsedPrecise/IrqlNotUsedPrecise.ql

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I believe this is in line with my previous comments, but why have a completely separate rule when we have built versioning into our queries? Let's keep it simple and only have one version at a time.

Reply:
In this case, it's because I built this extended version before learning that the must-flow analysis isn't working right now.  Rather than deal with keeping track of this version somewhere private, I figured I may as well include it as "experimental".  I could alternatively create a branch with this version as an update to the other rule, which would sit open until we could get updated QLPacks/etc.